openapi/products_spec.yaml

```diff
@@ -674,6 +674,8 @@ components:
         name:
           type: string
           example: A name for the product
+        pay_api_token:
```

Comment:
Is there any need to return this in the API? if not, maybe we can remove it from the response?

Reply:
Yeh it's used by selfservice in order to delete the API key when a payment link is deleted. Possibly we should be storing the token_link instead and deleting using that but that isn't how it currently works.
I just came across this when I was fixing some cypress stubbing and needed to use this in a stub and was a bit confused when this wasn't in the API spec.